modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -294,20 +306,20 @@ Creates a permission policy for a specified entity.
 |Name
 |Description
 |Type
-|Requirement
+|Presence
 
 |`entityReference`
-|Reference values of an entity including namespace and name
+|Reference values of an entity including kind, namespace, and name
```

Comment:
```suggestion
|Reference values of an entity including `kind`, `namespace`, and `name`
```

Reply:
We say `username` on line 48. Does that need to be changed to `name`?

Reply:
Are you talking about [this](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/3d76dd496563ed68d28275eb2918874d270dfe01/modules/admin/ref-rbac-rest-api-endpoints.adoc?plain=1#L46) mention of `username`?

or  was it the mention of `username` for the delete endpoints?

Reply:
Where we say _RBAC REST API endpoints, such as `/api/permission/policies/[kind]/[namespace]/[name]` for specified `kind`, `namespace`, and `username`, are the URI that you append to the base URL to access the corresponding resource._